packages/reporting-server/rest_server/repositories/test_parser_dispatcher.py

```diff
@@ -68,6 +71,12 @@ async def test_fleet_state_created(self):
         self.assertEqual(instance[1].fleet_name, "tinyRobot")
         self.assertEqual(instance[1].robot_name, "tinyRobot2")
 
+    async def test_task_summary_created(self):
+        data = 'task_summary:{"fleet_name": "tinyRobot", "task_id": "Loop0", "task_profile": {"task_id": "Loop0", "submission_time": {"sec": 131, "nanosec": 553000000}, "description": {"start_time": {"sec": 1623383402, "nanosec": 0}, "priority": {"value": 0}, "task_type": {"type": 1}, "station": {"task_id": "", "robot_type": "", "place_name": ""}, "loop": {"task_id": "", "robot_type": "", "num_loops": 1, "start_name": "supplies", "finish_name": "coe"}, "delivery": {"task_id": "", "items": [], "pickup_place_name": "", "pickup_dispenser": "", "pickup_behavior": {"name": "", "parameters": []}, "dropoff_place_name": "", "dropoff_ingestor": "", "dropoff_behavior": {"name": "", "parameters": []}}, "clean": {"start_waypoint": ""}}}, "state": 0, "status": "", "submission_time": {"sec": 0, "nanosec": 0}, "start_time": {"sec": 1623383362, "nanosec": 348338289}, "end_time": {"sec": 1623383449, "nanosec": 79154833}, "robot_name": "tinyRobot2"}\n'
```

Comment:
Maybe for a follow-up issue, we can add this information in a "mock" file.

Reply:
This is used in `test_task_summary_parser` as well. Can you please link the issue here?